### ISSUE TITLE
Account for VIES URL and subsequent call changing to https

### ISSUE THREAD

**Issue by ryanflach** [taxjar/vat_check#4]
Due to VIES updating to https, current calls to establish the client respond with:
```
<HTML><HEAD>
<TITLE>Redirect</TITLE>
</HEAD>
<BODY>
<FONT face="Helvetica">
<big><strong></strong></big><BR>
</FONT>
<blockquote>
<TABLE border=0 cellPadding=1 width="80%">
<TR><TD>
<FONT face="Helvetica">
<big>Redirect (policy_request_redirect)</big>
<BR>
<BR>
</FONT>
</TD></TR>
<TR><TD>
<FONT face="Helvetica">
Click <a href="https:&#x2F;&#x2F;ec.europa.eu&#x2F;taxation_customs&#x2F;vies&#x2F;checkVatService.wsdl">here</a> if you are not automatically redirected.
</FONT>
</TD></TR>
<TR><TD>
<FONT face="Helvetica">

</FONT>
</TD></TR>
<TR><TD>
<FONT face="Helvetica" SIZE=2>
<BR>
For assistance, contact your network support team.
</FONT>
</TD></TR>
</TABLE>
</blockquote>
</FONT>
</BODY></HTML>
```
I was able to spend a small amount of time looking at this, and it seemed like adding `follow_redirects: true` to https://github.com/taxjar/vat_check/blob/1898e5a3b64724165f809feb707a3a096fb57473/lib/vat_check/requests.rb#L6 got things working as expected through `vat_check.rb`, but I was unable to get the specs back to a passing state with this change.

**Comment by paulalden** [comment]
@fastdivision @aleccarper Is there any chance of getting this one patched? Seems like its breaking a clients production site as its 500'ing on submission of the VAT Codes. 

Thank you for the time you spend putting this gem together, its really appreciated.

**Comment by fastdivision** [comment]
Fixed in v1.0

**Comment by paulalden** [comment]
@fastdivision Thank you for taking the time to fix this. The Gem is really appreciated.
